packages/node-sdk/src/apis/DefaultApi.ts

```
@@ -373,6 +397,134 @@ export class DefaultApi extends runtime.BaseAPI {
+        "updateTargetRequest",
+        'Required parameter "updateTargetRequest" was null or undefined when calling setTarget().',
+      );
+    }
+
+    const queryParameters: any = {};
+
+    const headerParameters: runtime.HTTPHeaders = {};
+
+    headerParameters["Content-Type"] = "application/json";
+
+    if (this.configuration && this.configuration.apiKey) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)